staging/src/k8s.io/apiserver/plugin/pkg/audit/webhook/webhook.go

```diff
@@ -88,8 +89,7 @@ var (
 	//
 	// Can we make these passable to NewGenericWebhook?
 	groupFactoryRegistry = make(announced.APIGroupFactoryRegistry)
-	// TODO(audit): figure out a general way to let the client choose their preferred version
```

Comment:
It's just moved to a different place: https://github.com/kubernetes/kubernetes/pull/50824/files#diff-6b8b85c8f6acaf59bdf51e27a6b605bbR74